src/Runner.Worker/Handlers/StepHost.cs

```diff
@@ -58,18 +63,26 @@ public Task<string> DetermineNodeRuntimeVersion(IExecutionContext executionConte
             return Task.FromResult<string>(preferredVersion);
         }
 
-        public async Task<int> ExecuteAsync(string workingDirectory,
+        public async Task<int> ExecuteAsync(IExecutionContext context,
```

Comment:
why did we add context here?

Reply:
To give the FFManager access to FF vars given by the server. I'll change it to type `Variables` instead, from `context.Global.Variables` like discussed above.